AvgStarArray.jsx

```diff
@@ -0,0 +1,71 @@
+// LIBRARY IMPORTS
```

Comment:
Thanks so much for this! Could you file this under atelier > client > source > utilities?